charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -20,6 +20,7 @@
 - apiVersion: "v1"
   kindsRegexp: "^configmaps$"
   namespaceRegexp: "^cattle-fleet-|^fleet-"
+  excludeResourceNameRegexp: "fleet-agent*"
```

Comment:
Are you trying to exclude configmap in any namespace name matching "^cattle-fleet-|^fleet-" where the name is "fleet-agent"

That's the way I understand this rule

Reply:
Yes,  and for 2.9 only; that is at least what I aimed.

This is a screenshot on a backup from 2.9. I understood by excluding in this case the `fleet-agent.json` we would avoid this issue:
![image](https://github.com/user-attachments/assets/c770d37a-7f66-479c-9a65-03d2351a8a6a)

Content of `fleet-agent.json`:
```json
{"apiVersion":"v1","data":{"config":"{\"agentCheckinInterval\":\"0s\",\"labels\":{\"name\":\"local\",\"provider.cattle.io\":\"k3s\"},\"clientID\":\"cmf6rjvg9d6nc28kcj5sxhh8lfmz6g2c6xpw8xdm76cg56g8hgrm95\",\"bootstrap\":{},\"agentTLSMode\":\"strict\"}"},"kind":"ConfigMap","metadata":{"annotations":{"objectset.rio.cattle.io/applied":"H4sIAAAAAAAA/3yPvXKsMAyF30U1cLne5c9tqvQp3QgjFifGZixlZzLMvnvGkBRptpTOGX2fdphQEPQONobZ3UDDbgBvFORlIfvhwmsQSnf0BrSBmg0UBjyO5NmA3g0EXOnIfLS5VRjYUry7iVIFjwJWEvxlYAhRUFwMnMc4vpMVJqmSi5VFEU+Vi//cBBpmTyTlYVKOMQpLwq08S+UZHsSSv1hozSib6Dj+5lZiwXUDHT69L358nyEX5AU0XFpVq8vcNYq6+f/QD1M/to2iWg3jpGzb9EidvV4zLT/+1xPOJW9oc/LE9fEdAAD//6hbMZF4AQAA","objectset.rio.cattle.io/id":"fleet-agent-bootstrap-cattle-fleet-local-system"},"labels":{"objectset.rio.cattle.io/hash":"362023f752e7f1989d8b652e029bd2c658ae7c44"},"managedFields":[{"apiVersion":"v1","fieldsType":"FieldsV1","fieldsV1":{"f:data":{".":{},"f:config":{}},"f:metadata":{"f:annotations":{".":{},"f:objectset.rio.cattle.io/applied":{},"f:objectset.rio.cattle.io/id":{}},"f:labels":{".":{},"f:objectset.rio.cattle.io/hash":{}}}},"manager":"fleetcontroller","operation":"Update","time":"2024-07-23T07:38:42Z"}],"name":"fleet-agent","namespace":"cattle-fleet-local-system"}}
```
If I am mistaken in how to achieve it, I'd very welcome guidance in how to do so.

Reply:
ok dropping it. I discussed with team and this does not make sense.